database/abusescanner.go

```diff
@@ -16,6 +16,9 @@ import (
 )
 
 const (
+	// AbuseDatabaseName defines the name of the mongo database
+	AbuseDatabaseName = "abuse-scanner"
```

Comment:
We should probably be consistent with our naming. e.g. the accounts repo uses the `db` prefix.

Maybe `DBAbuseScanner` is better.

Reply:
done